### PR TITLE
FIX Backport numpy.isclose for np.version < 1.7

### PR DESCRIPTION
**Pull request by Anevar**
Attempt to fix issue #1938

### CODE REVIEW

**Comment by dengemann** [comment]
@Anevar LGTM! Does it solve your problem?

**Comment by eickenberg** [comment]
All but the last commit are buggy because we simply copied and pasted the numpy code for `isclose` into  `fixes.py`, and some of the expressions within the function had to be prefixed with `np` or `np.core`. The only 2 times this seems to be used is in `utils._time_mask`, so no idea whether such a huge backport is necessary, but OK.

**Comment by Anevar** [comment]
Yes now it Works!!! Thanks! :)

**Comment by eickenberg** [comment]
@Anevar happy to hear that! Congrats for the PR!

**Comment by dengemann** [comment]
Could you please make sure that the time mask related tests pass locally? Travis won't tell us.

**Comment by Anevar** [comment]
@eickenberg: Thank you! 

@dengemann: Yes they pass.

**Comment by agramfort** [comment]
LGTM

+1 for merge when travis is happy

**Comment by mainakjas** [comment]
Congrats @Anevar on your first PR! Looking forward to more :)

**Comment by Anevar** [comment]
Thanks! 
@mainakjas : I hope so too ;)

**Comment by eickenberg** [comment]
Travis failures due to doctests in `fixes.isclose`. I never know how to make them more lenient: The actual result is correct.

Also, no squashing needed?

**Comment by agramfort** [comment]
squash if you can.

**Comment by eickenberg** [comment]
@Anevar are you there tomorrow? We could finalize this then.

**Comment by Anevar** [comment]
Yes I will be there tomorrow. 

-----Original Message-----

File: mne/fixes.py [hunk 1]
```diff
@@ -631,3 +631,107 @@ def _sparse_block_diag(mats, format=None, dtype=None):
...
+        cond[~finite] = (x[~finite] == y[~finite])
+        if equal_nan:
+            # Make NaN == NaN
+            both_nan = np.isnan(x) & np.isnan(y)
+            cond[both_nan] = both_nan[both_nan]
+        return cond
+
+
+if LooseVersion(np.__version__) < LooseVersion('1.7'):
+   isclose = _isclose
```
Comment (agramfort) [inline mne/fixes.py]: mne/fixes.py:735:4: E111 indentation is not a multiple of four

**Comment by agramfort** [comment]
almost there...

**Comment by agramfort** [comment]
thanks

**Comment by eickenberg** [comment]
yay! congrats!

**Pull request merged**